Pilot/ErrorController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ErrorController.swift
+//  Pilot
+//
+//  Created by Nick Eckert on 8/12/16.
+//  Copyright © 2016 Sanction. All rights reserved.
+//
+
+import Cocoa
+
+class ErrorController: NSObject {
```

Comment:
I'm sure you're going to add more to this class but you should add functions like `displayError(error: String)` and `clearErrors()`, etc.